kcbq-connector/src/integration-test/java/com/wepay/kafka/connect/bigquery/it/BigQueryConnectorIntegrationTest.java

```diff
@@ -58,10 +58,12 @@
   public static final String KEYFILE_PROPERTY = "keyfile";
   public static final String PROJECT_PROPERTY = "project";
   public static final String DATASET_PROPERTY = "dataset";
+  public static final String KEYFILE_TYPE_PROPERTY = "keyFileType";
 
   private static String keyfile;
   private static String project;
   private static String dataset;
+  private static String keyFileType;
```

Comment:
This might be a confusing name. If this is supposed to signify whether a file will be used at all then maybe a better name for it exists, such as `keyType` or `keySource` or something similar. I'll leave references to this comment in other places this name exists.